### PR TITLE
Add ripemd family of hashes

### LINKED ISSUE #225: Implement ripemd160
Code: 0x1053

This hash function is used in Ethereum, Bitcoin, etc. Specifically, I need support for this hash function in the FVM.

### PR DESCRIPTION
**Pull request by mriise**
closes #225 
TODO tests

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -15,7 +15,7 @@ edition = "2018"
 resolver = "2"
 
 [features]
-default = ["std", "derive", "multihash-impl", "secure-hashes"]
+default = ["std", "derive", "multihash-impl", "secure-hashes", "ripemd"]
```
Comment (Stebalien) [inline Cargo.toml:18]: I wouldn't add it to the default list (for now).

**Review by Stebalien — APPROVED** [review]
lgtm

**Comment by Stebalien** [comment]
(modulo a test or two)

File: tests/lib.rs [hunk 2]
```diff
@@ -74,6 +86,28 @@ macro_rules! assert_encode {
    }
 }
 
+#[allow(unused)]
+#[test]
+/// prefix/multihash generating tool to aid when adding new tests
```
Comment (vmx) [inline tests/lib.rs:91]: This could also live in the `examples` directory.

**Review by vmx — APPROVED** [review]
As mentioned in the comment, I'd transform the utility function into an example. The rest looks good.

**Pull request merged**